Cert-manager E2E-test is failing

### Possible reason behind the error
Files with -values.yaml got [ignored](https://github.com/vmware-tanzu/community-edition/blob/2b55f0da0b712d3a40e028b9f44e540741eeeb33/.gitignore#L62-L63)
<!-- Before creating a new issue please search for any existing related [issues](https://github.com/vmware-tanzu/community-edition/issues?q=is%3Aissue+) before creating a new one. If you find an issue, join the discussion and add emoji or comments with your scenario or subscribe to follow updates. -->

## Bug Report

<!-- Please concisely describe the problem you are having. -->
On running E2E test for cert-manager it is failing with the error
```
make e2e-test
CGO_ENABLED=0 go run github.com/onsi/ginkgo/ginkgo -v e2e
Running Suite: cert-manager Package E2E Test Suite
==================================================
Random Seed: 1650888896
Will run 1 of 1 specs

STEP: installing cert-manager package
+ tanzu package available list -o json
[
  {
    "display-name": "App-Toolkit package for TCE",
    "latest-version": "0.1.0",
    "name": "app-toolkit.community.tanzu.vmware.com",
    "short-description": "Kubernetes-native toolkit to support application lifecycle"
  },
  {
    "display-name": "Cartographer",
    "latest-version": "0.3.0",
    "name": "cartographer.community.tanzu.vmware.com",
    "short-description": "Kubernetes native Supply Chain Choreographer."
  },
  {
    "display-name": "cert-injection-webhook",
    "latest-version": "0.1.1",
    "name": "cert-injection-webhook.community.tanzu.vmware.com",
    "short-description": "The Cert Injection Webhook injects CA certificates and proxy environment variables into pods"
  },
  {
    "display-name": "cert-manager",
    "latest-version": "1.8.0",
    "name": "cert-manager.community.tanzu.vmware.com",
    "short-description": "Certificate management"
  },
  {
    "display-name": "contour",
    "latest-version": "1.20.1",
    "name": "contour.community.tanzu.vmware.com",
    "short-description": "An ingress controller"
  },
  {
    "display-name": "external-dns",
    "latest-version": "0.10.0",
    "name": "external-dns.community.tanzu.vmware.com",
    "short-description": "This package provides DNS synchronization functionality."
  },
  {
    "display-name": "fluent-bit",
    "latest-version": "1.7.5",
    "name": "fluent-bit.community.tanzu.vmware.com",
    "short-description": "Fluent Bit is a fast Log Processor and Forwarder"
  },
  {
    "display-name": "Flux Source Controller",
    "latest-version": "0.21.2",
    "name": "fluxcd-source-controller.community.tanzu.vmware.com",
    "short-description": "The source-controller is a Kubernetes operator, specialised in artifacts acquisition from external sources such as Git, Helm repositories and S3 buckets."
  },
  {
    "display-name": "gatekeeper",
    "latest-version": "3.7.1",
    "name": "gatekeeper.community.tanzu.vmware.com",
    "short-description": "policy management"
  },
  {
    "display-name": "grafana",
    "latest-version": "7.5.11",
    "name": "grafana.community.tanzu.vmware.com",
    "short-description": "Visualization and analytics software"
  },
  {
    "display-name": "harbor",
    "latest-version": "2.4.2",
    "name": "harbor.community.tanzu.vmware.com",
    "short-description": "OCI Registry"
  },
  {
    "display-name": "knative-serving",
    "latest-version": "1.0.0",
    "name": "knative-serving.community.tanzu.vmware.com",
    "short-description": "Knative Serving builds on Kubernetes to support deploying and serving of applications and functions as serverless containers"
  },
  {
    "display-name": "kpack dependencies",
    "latest-version": "0.0.9",
    "name": "kpack-dependencies.community.tanzu.vmware.com",
    "short-description": "Dependencies in the form of Buildpacks and Stacks for the kpack package"
  },
  {
    "display-name": "kpack",
    "latest-version": "0.5.2",
    "name": "kpack.community.tanzu.vmware.com",
    "short-description": "kpack builds application source code into OCI compliant images using Cloud Native Buildpacks"
  },
  {
    "display-name": "Flux Kustomize Controller",
    "latest-version": "0.21.1",
    "name": "kustomize-controller.fluxcd.community.tanzu.vmware.com",
    "short-description": "Kustomize controller is one of the components in GitOps toolkit."
  },
  {
    "display-name": "local-path-storage",
    "latest-version": "0.0.22",
    "name": "local-path-storage.community.tanzu.vmware.com",
    "short-description": "This package provides local path node storage and primarily supports RWO AccessMode."
  },
  {
    "display-name": "multus-cni",
    "latest-version": "3.8.0",
    "name": "multus-cni.community.tanzu.vmware.com",
    "short-description": "This package provides the ability for enabling attaching multiple network interfaces to pods in Kubernetes"
  },
  {
    "display-name": "prometheus",
    "latest-version": "2.27.0-1",
    "name": "prometheus.community.tanzu.vmware.com",
    "short-description": "A time series database for your metrics"
  },
  {
    "display-name": "velero",
    "latest-version": "1.8.0",
    "name": "velero.community.tanzu.vmware.com",
    "short-description": "Disaster recovery capabilities"
  },
  {
    "display-name": "whereabouts",
    "latest-version": "0.5.1",
    "name": "whereabouts.community.tanzu.vmware.com",
    "short-description": "A CNI IPAM plugin that assigns IP addresses cluster-wide"
  }
]
+ tanzu package available list cert-manager.community.tanzu.vmware.com -o json
[
  {
    "name": "cert-manager.community.tanzu.vmware.com",
    "released-at": "2021-08-23 22:52:51 +0530 IST",
    "version": "1.5.4"
  },
  {
    "name": "cert-manager.community.tanzu.vmware.com",
    "released-at": "2021-08-23 22:52:51 +0530 IST",
    "version": "1.5.5"
  },
  {
    "name": "cert-manager.community.tanzu.vmware.com",
    "released-at": "2021-10-29 17:30:00 +0530 IST",
    "version": "1.6.1"
  },
  {
    "name": "cert-manager.community.tanzu.vmware.com",
    "released-at": "2021-10-29 17:30:00 +0530 IST",
    "version": "1.6.3"
  },
  {
    "name": "cert-manager.community.tanzu.vmware.com",
    "released-at": "2021-10-29 17:30:00 +0530 IST",
    "version": "1.7.2"
  },
  {
    "name": "cert-manager.community.tanzu.vmware.com",
    "released-at": "2021-10-29 17:30:00 +0530 IST",
    "version": "1.8.0"
  }
]
+ tanzu package install cert-manager --poll-interval 10s --poll-timeout 20m --namespace default --package-name cert-manager.community.tanzu.vmware.com --version 1.8.0 --values-file fixtures/cert-manager-values.yaml
Error: failed to read from data values file 'fixtures/cert-manager-values.yaml': open fixtures/cert-manager-values.yaml: no such file or directory
Error: exit status 1

Failure [4.149 seconds]
[BeforeSuite] BeforeSuite 
/Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/cert-manager/1.8.0/test/e2e/cert_manager_suite_test.go:47

  Unexpected error:
      <*errors.errorString | 0xc00005a0c0>: {
          s: "Error: failed to read from data values file 'fixtures/cert-manager-values.yaml': open fixtures/cert-manager-values.yaml: no such file or directory\nError: exit status 1\n\nexit status: 1",
      }
      Error: failed to read from data values file 'fixtures/cert-manager-values.yaml': open fixtures/cert-manager-values.yaml: no such file or directory
      Error: exit status 1
      
      exit status: 1
  occurred

  /Users/amansha/go/src/github.com/vmware-tanzu/community-edition/addons/packages/cert-manager/1.8.0/test/e2e/cert_manager_suite_test.go:131
------------------------------
STEP: cleaning up cert-manager package
+ tanzu package installed delete cert-manager --poll-interval 10s --poll-timeout 20m --namespace default --yes

STEP: validating the cert-manager package install no longer exists
+ kubectl -n default get packageinstall cert-manager
Error from server (NotFound): packageinstalls.packaging.carvel.dev "cert-manager" not found

STEP: cleaning up cert-manager namespace
+ kubectl delete namespace cert-manager
Error from server (NotFound): namespaces "cert-manager" not found


Ran 1 of 0 Specs in 4.919 seconds
FAIL! -- 0 Passed | 1 Failed | 0 Pending | 0 Skipped
--- FAIL: Test (4.92s)
FAIL

Ginkgo ran 1 suite in 6.656836347s
Test Suite Failed
exit status 1
make: *** [e2e-test] Error 1

```
## Expected Behavior

<!-- Describe what you expected to happen. -->

## Steps to Reproduce the Bug
```
cd addons/packages/cert-manager/1.8.0/test && make e2e-test
```
<!-- Describe steps to reproduce here and how reproducible this issue is (occasionally, often, always) -->

## Screenshots or additional information and context

<!-- If screenshots or snippets can be provided, please do so here. Otherwise please add additional context if necessary. -->

## Environment Details

* Build version (`tanzu version`): `v0.11.4`
* Deployment (Managed/Unmanaged cluster): UnmanagedCluster
* Infrastructure Provider (Docker/AWS/Azure/vSphere): Docker
* Operating System (client): Mac

## Diagnostics and log bundle
<!-- Collecting diagnostics and log bundles can help troubleshoot your issue -->

--- a/addons/packages/cert-manager/1.7.2/test/e2e/cert_manager_suite_test.go
+++ b/addons/packages/cert-manager/1.7.2/test/e2e/cert_manager_suite_test.go
@@ -55,7 +55,7 @@ var _ = BeforeSuite(func() {
 
 	version := findPackageAvailableVersion(packageName, "1.7.2")
 
-	valuesFilename := filepath.Join("fixtures", "cert-manager-values.yaml")
+	valuesFilename := filepath.Join("fixtures", "values.yaml")
 	installPackage(packageInstallName, packageName, version, valuesFilename)
 
 	By("validating cert-manager package is reconciled")

--- a/addons/packages/cert-manager/1.7.2/test/e2e/fixtures/values.yaml
+++ b/addons/packages/cert-manager/1.7.2/test/e2e/fixtures/values.yaml
@@ -1,0 +1,1 @@
+namespace: cert-manager

--- a/addons/packages/cert-manager/1.8.0/test/e2e/cert_manager_suite_test.go
+++ b/addons/packages/cert-manager/1.8.0/test/e2e/cert_manager_suite_test.go
@@ -55,7 +55,7 @@ var _ = BeforeSuite(func() {
 
 	version := findPackageAvailableVersion(packageName, "1.8.0")
 
-	valuesFilename := filepath.Join("fixtures", "cert-manager-values.yaml")
+	valuesFilename := filepath.Join("fixtures", "values.yaml")
 	installPackage(packageInstallName, packageName, version, valuesFilename)
 
 	By("validating cert-manager package is reconciled")

--- a/addons/packages/cert-manager/1.8.0/test/e2e/fixtures/values.yaml
+++ b/addons/packages/cert-manager/1.8.0/test/e2e/fixtures/values.yaml
@@ -1,0 +1,1 @@
+namespace: cert-manager

